package.json

```diff
@@ -12,7 +12,9 @@
     "test": "cross-env BABEL_ENV=cjs jest --coverage",
     "test:watch": "npm test -- --watch",
     "release": "node scripts/release.js",
-    "postinstall": "node scripts/installNestedPackageDeps.js"
+    "postinstall": "node scripts/installNestedPackageDeps.js",
+    "format": "rm -rf src/packages/*/node_modules;prettier --semi false --trailing-comma es5 --single-quote --write '{src/,scripts}/**/*.js'",
```

Comment:
It is a little inconvenient if `node_modules` have been removed per commit, we need to re-install it while developing later. How about adjusting pattern something like:

```json
"format": "prettier --semi false --trailing-comma es5 --single-quote --write 'scripts/*.js' 'src/packages/*/*.js' 'src/packages/*/!(node_modules)/**/*.js'",

```